src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1977,6 +1977,34 @@ public void xclaimJustId(byte[] key, byte[] groupname, byte[] consumername, long
     xclaim(key, groupname, consumername, minIdleTime, params, ids, true);
   }
 
+  public void xautoclaim(byte[] key, byte[] groupName, byte[] consumerName,
+      long minIdleTime, byte[] start, XAutoClaimParams params) {
+    xautoclaim(key, groupName, consumerName, minIdleTime, start, params, false);
+  }
+
+  public void xautoclaim(byte[] key, byte[] groupName, byte[] consumerName,
+      long minIdleTime, byte[] start, XAutoClaimParams params, boolean justId) {
```

Comment:
```suggestion
  private void xautoclaim(byte[] key, byte[] groupName, byte[] consumerName,
      long minIdleTime, byte[] start, XAutoClaimParams params, boolean justId) {
```

Reply:
Changed the access specifier to private for this internal method